config/locales/en.yml

```diff
@@ -1656,7 +1656,7 @@ en:
     errors:
       code_401: "You need to be authenticated to access this resource."
       code_403: "You are not authorized to access this resource."
-      code_404: "Couldn\'t find %{type} with id=%{id}"
```

Comment:
The point is, that it is a lie to have this localization.

In fact the old code just used whatever message was in `RecordNotFound`. In places where we did not rescue from `RecordNotFound` we really used this localization.

This had two immediate effects:

1. No localization
2. If you changed the localization (because you could), the tests would immediately fail because they *assume* that the localization is used, when it is in fact not.

There is also non intuitive behaviour with regards to punctuation.

As `ActiveRecord` does not allow to infere real information from the `RecordNotFound` error, it is not possible to keep a specialized error message. We thus decided to simply generalize it. It should be good enough and at least not flawed by design.